src/common/models/repo.go

```diff
@@ -37,3 +37,12 @@ type RepoRecord struct {
 func (rp *RepoRecord) TableName() string {
 	return RepoTable
 }
+
+// RepositoryQuery : query parameters for repository
+type RepositoryQuery struct {
+	Name        string
+	ProjectIDs  []int64
+	ProjectName string
+	LabelIDs    []int64
+	Pagination
```

Comment:
The usage of `Pagination` in `RepositoryQuery` and `Label` is not consistent with other models, such as `project` `user`